### PR TITLE
Update delegate.md

### PR DESCRIPTION
**Pull request by kat-enos**
Release notes for 2/23 SaaS drop,

# Harness Developer Pull Request
Thanks for helping us make the Developer Hub better. The PR will be looked at
by the maintainers. 

## What Type of PR is This?

- [ ] Issue
- [ ] Feature
- [ x] Maintenance/Chore

If tied to an Issue, list the Issue(s) here:

* *Issue(s)*

## House Keeping
Some items to keep track of. Screen shots of changes are optional but would help the maintainers review quicker. 

- [ ] Tested Locally
- [ ] *Optional* Screen Shoot.

### CODE REVIEW

File: release-notes/delegate.md [hunk 1]
```diff
@@ -14,6 +14,28 @@ Harness deploys changes to Harness SaaS clusters on a progressive basis. This me
...
+The repository location of the Helm chart for the NextGen delegate is changing. (DEL-5576)
+
+The repository at https://app.harness.io/storage/harness-download/delegate-helm-chart/ is being deprecated. The Helm chart will no longer be available from the repository at https://app.harness.io/storage/harness-download/delegate-helm-chart/. To ensure retrieval of the most recent Helm chart, update your repository references to https://app.harness.io/storage/harness-download/harness-helm-charts/.
+:::
+
+### What's new
+
+This release introduces the following new features and enhancements:
+
+-  - The delegate hostname can be used for dynamic selection. You can select a delegate from those in your delegate groups during pipeline runtime. (DEL-5052)
```
Comment (pratmit) [inline release-notes/delegate.md:32]: The delegate hostname can be used for dynamic selection. You can select a delegate hostname from those in your delegate groups during pipeline runtime.

**Review by pratmit — CHANGES_REQUESTED** [review]

File: release-notes/delegate.md (hunk 1 above)
Comment (kat-enos) [reply release-notes/delegate.md:32]: @pratmit, they're selecting the hostname of one delegate from the delegates in the delegate groups, aren't they? Your suggestion means they can select a delegate hostname from the hostnames in their delegate groups. Do you mean that the delegates are listed by hostname? Please adviser.

**Review by ravilach — COMMENTED** [review]
@kat-enos don't forget to change the publish date in the front matter please :-)

File: release-notes/delegate.md (hunk 1 above)
Comment (pratmit) [reply release-notes/delegate.md:32]: yes delegate is listed by hostname. Please refer the ticket if more clarification is needed.
This is the public release notes which is also fine by me - "User can provide delegate hostname as a dynamic selector to pick one individual delegate among the delegate groups"

Comment (kat-enos) [reply release-notes/delegate.md]: @pratmit This one took the same change as the CG one -- the change we agreed to. That was slipped in. yesterday, I believe. I am resolving this conversation.

**Comment by kat-enos** [comment]
@ravilach Front matter corrected yesterday. Cheers.

File: release-notes/delegate.md [hunk 2]
```diff
@@ -14,6 +14,28 @@ Harness deploys changes to Harness SaaS clusters on a progressive basis. This me
 
 For FirstGen SaaS release notes, see [Harness SaaS Release Notes (FirstGen)](/docs/first-gen/firstgen-release-notes/harness-saa-s-release-notes). For Self-Managed Enterprise Edition, see [Self-Managed Enterprise Edition release notes](/release-notes/self-managed-enterprise-edition).
 
+## February 23, 2023, version 785xx
+### Delegate version 78500
+
+Harness NextGen release 785xx includes the following changes for Harness Delegate.
```
Comment (aimurphy) [inline release-notes/delegate.md:20]: I think it needs the final version number - line 17 and 20?

**Review by aimurphy — DISMISSED** [review]
lgtm with 1 comment

**Review by aimurphy — APPROVED** [review]

File: release-notes/delegate.md (hunk 2 above)
Comment (kat-enos) [reply release-notes/delegate.md]: Done.

**Pull request merged**